SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
```

Comment:
This is assigned to someone else i believe here #2

Reply:
I'm sorry about that. Just a moment, and I can resolve this